src/v2/Apps/Fair/Components/FairHeader/FairHeader.tsx

```diff
@@ -1,170 +1,55 @@
 import React from "react"
-import {
-  Box,
-  BoxProps,
-  GridColumns,
-  Column,
-  Flex,
-  Text,
-  Spacer,
-  Join,
-} from "@artsy/palette"
+import { Box, BoxProps, Flex, Text } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
+import { FairHeaderIconFragmentContainer } from "./FairHeaderIcon"
 import { FairHeader_fair } from "v2/__generated__/FairHeader_fair.graphql"
 import { FairTimingFragmentContainer as FairTiming } from "./FairTiming"
-import { FairHeaderIconFragmentContainer } from "./FairHeaderIcon"
-import {
-  FairEditorialFragmentContainer,
-  FAIR_EDITORIAL_AMOUNT,
-} from "../FairEditorial"
-import { FairCollectionsFragmentContainer } from "../FairCollections"
-import { FairFollowedArtistsFragmentContainer } from "../FairFollowedArtists"
-import { useSystemContext } from "v2/System"
-import { RouterLink } from "v2/System/Router/RouterLink"
-import { InfoSection } from "v2/Components/InfoSection"
+import { Media } from "v2/Utils/Responsive"
 
 interface FairHeaderProps extends BoxProps {
   fair: FairHeader_fair
 }
 
 const FairHeader: React.FC<FairHeaderProps> = ({ fair }) => {
-  const {
-    name,
-    about,
-    tagline,
-    location,
-    ticketsLink,
-    hours,
-    links,
-    contact,
-    summary,
-    tickets,
-  } = fair
-
-  const { user } = useSystemContext()
-  const hasArticles = (fair.articlesConnection?.edges?.length ?? 0) > 0
-  const hasCollections = (fair.marketingCollections?.length ?? 0) > 0
+  const { name } = fair
 
   return (
-    <Box>
-      <GridColumns mt={[2, 6]}>
-        <Column span={6}>
-          <Flex mb={2}>
-            <FairHeaderIconFragmentContainer fair={fair} mr={2} />
-            <Box>
-              <Text as="h1" variant="xl">
-                {name}
-              </Text>
-
-              <FairTiming fair={fair} />
-            </Box>
-          </Flex>
-        </Column>
-
-        <Column span={6}>
-          <Join separator={<Spacer mt={2} />}>
-            <Text variant="md" textTransform="uppercase">
-              About
+    <>
+      {/* Desktop Fair Header */}
+      <Media greaterThan="xs">
```

Comment:
**suggestion**: We usually use this `Media` component when the layout between sizes differ greatly, but in this case, I think that we should be able to get by using [responsive props](https://palette.artsy.net/guides/responsive/) so that we're not rendering the same content twice:

Would you mind taking a moment to see if we can consolidate this JSX and move from `Media` to responsive props where possible?

```jsx
<Flex my={[2]}>
  <FairHeaderIconFragmentContainer fair={fair} mr={2} />
  <Box display={["flex", "block"] flexDirection="column" justifyContent="center" >
    <Text as="h1" variant={["lg", "xl"]}>
      {name}
    </Text>
    <FairTiming fair={fair} />
  </Box>
</Flex>
```